mumd/src/audio.rs

```diff
@@ -354,3 +381,21 @@ impl Audio {
         play_sounds.extend(samples.iter().skip(l));
     }
 }
+
+// moo
+fn get_sfx(file: &str) -> Cow<'static, [u8]> {
```

Comment:
This might be more descriptive to call `path` instead. Also, instead of taking a `&str`, consider taking a `P` where `<P: AsRef<Path>>`.

Reply:
Resolved through discussion:

We actually take a file name as a string that is looked for in different places.